scapy/tools/UTscapy.py

```diff
@@ -843,6 +843,9 @@ def main(argv):
             from scapy.arch.windows import route_add_loopback
             route_add_loopback()
 
+        # Add SCAPY_ROOT_DIR environment variable, used for tests
+        os.environ['SCAPY_ROOT_DIR'] = os.environ.get("PWD", os.getcwd())
```

Comment:
Shall we remove this variable at the end of UTScapy execution?

Reply:
Done

Reply:
Why would we do that? The environment variable won't be passed to other programs anyway when UTScapy terminates. Or did I miss something?

Reply:
It’s cleaner, as people may use UTscapy outside travis

Reply:
I do not agree. You do not remove an environment variable set by the user.